packages/orchestrator-ui-components/src/icons/WfoCheckmarkCircleFill.tsx

```diff
@@ -14,7 +14,6 @@ export const WfoCheckmarkCircleFill: FC<WfoIconProps> = ({
         version="1.1"
         xmlns="http://www.w3.org/2000/svg"
     >
-        <title>icon/checkmark-circle-fill</title>
```

Comment:
Adjusting SVG files might not be the best solution. Can't you apply a css rule?

Reply:
I don't think it is possible only with CSS. I've tried before but didn't succeed. 

And if there is a way with css, we always have to hide the default tooltip (title). Isn't it better to just remove it from the svg ?